### PR TITLE
#8 Added SelectedTitle property

### PR DESCRIPTION
**Pull request by eatskolnikov**
**What's new?**

- Added public access to SelectedTitle property

Closes: #8

### CODE REVIEW

**Review by christianhelle — APPROVED** [review]
This looks fine

**Comment by eatskolnikov** [comment]
@christianhelle does the CI automatically publishes the nuget package?

File: src/Controls/ScrollableSegmentedControl.cs [hunk 1]
```diff
@@ -56,6 +55,16 @@ public ScrollableSegmentedControl(IEnumerable<UIImage> sectionImages, IEnumerabl
 
         public Func<ScrollableSegmentedControl, string, int, bool, NSAttributedString> TitleFormatter { get; set; }
 
+
+        public string SelectedTitle
+        {
+            get
+            {
+                if (sectionTitles.Count > selectedIndex)
+                    return sectionTitles[selectedIndex];
```
Comment (christianhelle) [inline src/Controls/ScrollableSegmentedControl.cs:64]: Build fails as `selectedIndex` is missing

**Review by christianhelle — CHANGES_REQUESTED** [review]
Please fix the build error and I will merge this in

File: src/Controls/ScrollableSegmentedControl.cs (hunk 1 above)
Comment (eatskolnikov) [reply src/Controls/ScrollableSegmentedControl.cs]: Weird! It compiled on my machine, I have submitted a change for this.

**Comment by christianhelle** [comment]
> @christianhelle does the CI automatically publishes the nuget package?

No, I manually start a release from Azure Pipelines. I had a continuous delivery setup previously but I once had some garbage merged into master and that automatically got published to nuget.org so I disabled auto publishing

**Comment by eatskolnikov** [comment]
> > @christianhelle does the CI automatically publishes the nuget package?
> 
> No, I manually start a release from Azure Pipelines. I had a continuous delivery setup previously but I once had some garbage merged into master and that automatically got published to nuget.org so I disabled auto publishing

Seems like a strategy, cool!

**Comment by christianhelle** [comment]
Hmmm the build still fails

https://dev.azure.com/christianhelle/ScrollableSegmentedControl/_build/results?buildId=2553&view=logs&j=275f1d19-1bd8-5591-b06b-07d489ea915a&t=298bf322-c6d7-5231-9614-c0e9bc17313e

Do you have any problems building from your machine? I probably need to make some updates to the build pipeline since I haven't touched this project in 16 months

**Comment by christianhelle** [comment]
Your changes should be good enough. I'll fix the build pipelines from another branch

**Pull request merged**